### DISCUSSION TITLE
Auth guard [null] is not defined.

### DISCUSSION THREAD

**Discussion by blondie63** [Laravel-Backpack/community-forum#936] [Q&A (Help)] [answered]
On my last prj i've installed add-on Permission Manager following **all steps** but i've this err:
<img width="1406" alt="bkp6x-perm-man-issue" src="https://github.com/Laravel-Backpack/community-forum/assets/1261851/45b548b8-d57a-451a-b8e1-a0fbdcfb8942">

Some help ? :)
Thanks

**Comment by blondie63** [comment, marked as answer]
Sorry i've found my mistake !!
I missed point No. 7 that 7a and 7b were alternative in Readme !!

**Comment by emjayess** [comment]
> I missed point No. 7 that 7a and 7b were alternative in Readme !!

What is this referring to, and what does this mean? Please expound..

Backpack's `config/backpack/base.php` configuration reads as follows:
```
    // The guard that protects the Backpack admin panel.
    // If null, the config.auth.defaults.guard value will be used.
    'guard' => 'null',
```

Yet, as soon as I adjust this, I'm staring right at this same error. 🤷🏼

**Reply by pxpm** [reply]
Hey @emjayess `null !== 'null'`. 

You are telling backpack that the guard name is "null", not that there is no guard (use the laravel default) by setting the config value to `null`. 

Sum up, don't wrap `null` as if it were a string. 

Let me know if that helps. 

Cheers

**Reply by emjayess** [reply]
> Hey @emjayess `null !== 'null'`.
> 
> You are telling backpack that the guard name is "null", not that there is no guard (use the laravel default) by setting the config value to `null`.
> 
> Sum up, don't wrap `null` as if it were a string.
> 
> Let me know if that helps.
> 
> Cheers

# 😳

.. thank you, I'll, uh, see myself out.
